src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -30,23 +47,46 @@ const defaultProps = {
 };
 
 const IOUAmountPage = props => (
-    <View style={styles.pageWrapper}>
+    <View style={[styles.flex1, styles.pageWrapper]}>
         {props.iou.loading && <ActivityIndicator color={themeColors.text} />}
-        <TouchableOpacity
-            style={[styles.button, styles.w100, styles.mt5]}
-            onPress={props.onStepComplete}
-        >
-            <Text style={[styles.buttonText]}>
-                Next
-            </Text>
-        </TouchableOpacity>
+        <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter]}>
+            {props.isSmallScreenWidth
+                ? <Text style={styles.iouAmountText}>{props.selectedCurrency + props.amount}</Text>
+                : (
+                    <TextInput
+                            style={styles.iouAmountTextInput}
+                            onKeyPress={event => props.numberPressed(event.key)}
+                            value={props.selectedCurrency + props.amount}
```

Comment:
Sorry this wasn't clearer, but could we keep the currency value in a separate Text Component? This is because it will become touchable with future changes (and will let the user change currency).

Reply:
When I use Text and TextInput components together, styling doesn't work great since TextInput doesn't grow with content like Text component. I think we have 2 options here;
1) Divide view by half and align currency to the right and TextInput to the left. When user enters amount, currency view won't move left.
<img width="200" alt="Screenshot 2021-03-15 at 22 24 49" src="https://user-images.githubusercontent.com/3853003/111229209-6b560c00-85dd-11eb-91d3-2a90ad3203ec.png"> <img width="200" alt="Screenshot 2021-03-15 at 22 25 03" src="https://user-images.githubusercontent.com/3853003/111229213-6beea280-85dd-11eb-8f65-22ce4cce30d7.png">

2) Implement a AutoGrowTextInput component that adjusts TextInput width dynamically by creating an invisible Text view and use its width to set TextInput's width. Something like this https://stackoverflow.com/a/30206471

Reply:
Ah thanks for explaining. Once we're using the currency symbol the offset wouldn't be as bad as the screenshot above, but I still think we'll need to go with option 2 to avoid an uncentered amount.

Reply:
I updated `TextInput` to have a dynamic width based on content using a hidden `Text` view. Now, it looks like in this video. It is now centered like mobile.

https://user-images.githubusercontent.com/3853003/111378515-ce0cdd80-8699-11eb-8bbd-adc1f1d53d76.mov